docs/source/index.rst

```diff
@@ -114,18 +114,18 @@ Please see our `contributing guidelines <https://github.com/cleanlab/cleanlab/bl
    :hidden:
    :caption: Tutorials
 
-   tutorials/indepth_overview
-   tutorials/image
-   tutorials/text
-   tutorials/tabular
-   tutorials/audio
-   tutorials/dataset_health
-   tutorials/outliers
-   tutorials/multiannotator
-   tutorials/multilabel_classification
-   tutorials/token_classification
-   tutorials/pred_probs_cross_val
-   tutorials/faq
+   Getting Started With Cleanlab <tutorials/indepth_overview>
+   Image Classification (Pytorch) <tutorials/image>
+   Text Classification (Tensorflow) <tutorials/text>
+   Tabular Classification     (Scikit-Learn) <tutorials/tabular>
+   Audio Classification (Speechbrain) <tutorials/audio>
+   Find Dataset-level Issues <tutorials/dataset_health>
+   Identifying Outliers (Pytorch) <tutorials/outliers>
+   Improving Consensus Labels for Multiannotator Data <tutorials/multiannotator>
+   Multi-Label Classification (Scikit-Learn) <tutorials/multilabel_classification>
+   Token Classification <tutorials/token_classification>
```

Comment:
```suggestion
   Token Classification (text) <tutorials/token_classification>
```

Reply:
only suggesting ^ if it still fits on one line, otherwise let's keep this as is.